src/Context/SilverStripeContext.php

```diff
@@ -276,6 +277,27 @@ public function before(BeforeScenarioScope $event)
         }
     }
 
+    /**
+     * @AfterStep
+     *
+     * Wait for all requests to be handled after each step
+     *
+     * @param AfterStepScope $event
+     */
+    public function waitResponsesAfterStep(AfterStepScope $event)
+    {
+        $success = $this->testSessionEnvironment->waitForPendingRequests();
+        if (!$success) {
+            echo (
```

Comment:
Nit picking, but echo is a construct rather than a function, so doesn't need brackets

Reply:
Sure, I just assumed it would look tidier if all the lines aligned with the same indentation